perception/lidar_centerpoint_tvm/lib/postprocess/generate_detected_boxes.cpp

```diff
@@ -69,39 +69,39 @@ void generateBoxes3D_worker(
   // heatmap: N = class_size, offset: N = 2, z: N = 1, dim: N = 3, rot: N = 2, vel: N = 2
   for (std::size_t idx = 0; idx < grids_per_thread; idx++) {
     std::size_t grid_idx = thread_idx * grids_per_thread + idx;
+    const auto grid_size = config.grid_size_y_ * config.grid_size_x_;
     const auto down_grid_size = config.down_grid_size_y_ * config.down_grid_size_x_;
     if (grid_idx >= down_grid_size) {
       return;
     }
 
-    const auto yi = grid_idx / config.down_grid_size_x_;
-    const auto xi = grid_idx % config.down_grid_size_x_;
+    const auto yi = config.downsample_factor_ * (grid_idx / config.down_grid_size_x_);
+    const auto xi = config.downsample_factor_ * (grid_idx % config.down_grid_size_x_);
+    std::size_t array_idx = yi * config.grid_size_x_ + xi;
```

Comment:
Multiplying downsample factor with idx gives discrete values of xi and yi, which does not seem consistent with the original implementation. 

https://github.com/autowarefoundation/autoware.universe/blob/b78b17d6be85611e0686b1c319f3ae0ffe7bd890/perception/lidar_centerpoint/lib/postprocess/postprocess_kernel.cu#L66-L69